startup.sh

```diff
@@ -32,12 +32,31 @@ function handleStartup() {
     if [ ! -z "$ADMINPASS" ] && [ ! -a "$ADMINMAIL" ]; then
       /opt/kimai/bin/console kimai:create-user superadmin $ADMINMAIL ROLE_SUPER_ADMIN $ADMINPASS
     fi
+    # I think kimai:install overwrites the monolog config
+    if [ "$APP_ENV" == "dev" ]; then
+      cat <<EOF > /opt/kimai/config/packages/dev/monolog.yaml
+monolog:
+  handlers:
+    main:
+      type: error_log
+      level: debug
+EOF
+    else 
+      cat <<EOF > /opt/kimai/config/packages/prod/monolog.yaml
+monolog:
+  handlers:
+    main:
+      type: error_log
+      level: error
+EOF
+    fi
     echo $KIMAI > /opt/kimai/installed
   fi
   echo "Kimai2 ready"
 }
 
 function runServer() {
+  /opt/kimai/bin/console cache:clear --env=$APP_ENV
```

Comment:
either add a `/opt/kimai/bin/console cache:warmup --env=$APP_ENV` or use the reload command `/opt/kimai/bin/console kimai:reload --env=$APP_ENV`

Reply:
Done